lib/index.js

```diff
@@ -13,12 +15,20 @@ const linterUiDefault = {
       atom.packages.getLoadedPackage('linter-ui-default').metadata['package-deps'].push('busy-signal')
     }
 
-    if (!atom.inSpecMode()) {
-      // eslint-disable-next-line global-require
-      require('atom-package-deps').install('linter-ui-default', true)
+    let callbackID
```

Comment:
There would be no need to create this extra variable, because this is valid

```
const something = setInterval(function() {
  clearInterval(something)
})
```